helm/trivy/templates/statefulset.yaml

```
@@ -36,6 +36,10 @@ spec:
       securityContext:
 {{ toYaml .Values.podSecurityContext | indent 8 }}
       {{- end }}
+      {{- if .Values.image.pullSecret }}
+      imagePullSecrets:
+        - name: {{- if .Values.image.pullSecret }}
```

Running template command failed in my env:

```
helm template trivy ./helm/trivy \
  --debug --namespace trivy \
  --set image.pullSecret=my-secret
```

I guess this is due to the syntax error?

```suggestion
        - name: {{ .Values.image.pullSecret }}
```

---

Thanks for the catch. This was a Copy&Paste error.